staging/public-swarm.yaml

```diff
@@ -1,25 +1,68 @@
+bootnode:
+  enabled: true
+  nodeSelector:
+    nospot: "true"
+  tolerations:
+    - key: "nospot"
+      operator: "Exists"
+      effect: "NoSchedule"
+    - key: "nospot"
+      operator: "Exists"
+      effect: "NoExecute"
+  terminationGracePeriodSeconds: 3
+  updateStrategy:
+    type: "OnDelete"
+  config:
+    extraFlags:
+      - --ws
+      - --wsaddr=0.0.0.0
+      - --wsorigins=*
+      - --wsapi=admin,net,debug,bzz,stream,accounting
+      - --sync-mode=all
```

Comment:
Not sure we need any of these extra flags for bootnode?

Reply:
Good catch. It was supposed to be on the swarm side. Fixed!